evil-owl.el

```diff
@@ -155,17 +155,23 @@ show, and outputs an entry string (newline included)."
    groups
    evil-owl-separator))
 
+(defun evil-owl--trim-contents (contents)
+  "Return a string no more than `evil-owl-register-char-limit' characters in length."
+  (replace-regexp-in-string
+   "\n" "^J"
+   (if (and evil-owl-register-char-limit
```

Comment:
Since the character limit applies to both marks and registers now, I'd suggest renaming the limit variable to `evil-owl-text-limit` (or a maybe a better name?).  To not surprise people who've already downloaded `evil-owl` though, you should also do `(defalias 'evil-owl-register-char-limit 'evil-owl-text-limit)` and mark `evil-owl-register-char-limit` as obsolete.